src/gt4py/frontend/gtscript_frontend.py

```diff
@@ -1258,7 +1378,14 @@ def run(self):
         )
         main_func_node = self.ast_root.body[0]
 
-        assert hasattr(self.definition, "_gtscript_")
+        if not hasattr(self.definition, "_gtscript_"):
```

Comment:
I added this so that I could test the frontend parsing independently from the backend generation (using the gt4py.stencil decorator). I found it useful, but I admit it's more of a patch to make it work for now.